tests/testsuite/kubevirtresource.go

```diff
@@ -51,6 +55,28 @@ var (
 	originalKV            *v1.KubeVirt
 )
 
+// GetVMGuestByArchitecture returns the VMGuest based on the Architecture.
+func GetVMGuestByArchitecture(opts ...libvmi.Option) (*v1.VirtualMachineInstance,
+	console.LoginToFunction, bool) {
+	var vmi *v1.VirtualMachineInstance
+	var loginToVMI console.LoginToFunction
+	var isAlpine bool
+	arch := putil.TranslateBuildArch()
+	// Cirros is not supported for s390x.
+	// Use Alpine instead of Cirros for s390x.
```

Comment:
If this is the case shouldn't we just deprecate the use of CirrOS in our tests and move everything across to Alpine?

Reply:
Dropping cirros in favour of alpine would simplify our enabling e2e test for s390x, but i am not sure if there would be consequences for the other arches.
Might be a good discussion topic on the mailing list/next community meeting.

Reply:
I can do this way. But it might cause confusion. Because when someone reads the code, and they came across NewCirros in the test they will think that it will create a Cirros VMI. But when they run the test on the s390x cluster. It will create an Alpine VMI.

Reply:
Yup fair points. Can you still push ahead with the discussion around the removal of cirros on the ML?

Reply:
Discussing this offline it might even be better to skip the offending `CirrOS` tests for now, avoiding the need for this code and port them over to `Alpine` to allow them to run on `s390x` eventually.